icons/svg/call-voicemail_18.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Are theses icons related to this update?

Reply:
No, my branch is just for date range picker component. I did not change or add any icons. That may be from master branch.

Reply:
Can you `git pull --rebase` on your branch against the master branch? That should clear up these extra commits.

Reply:
Ok, got it.